### PR TITLE
Added mail sending with mandrill

### PR DESCRIPTION
**Pull request by rominariedel**
# Summary

- Added mandrill configuration
- A mail with the last ten articles is sent when clicking "Send me more!" button displayed in header.

# Trello Card

https://trello.com/c/fLfJpJ2B/14-crear-email-con-los-ultimos-10-articulos-creados

# Screenshots

-
print1
-
print2

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
 development:
   secret_key_base: b8b615677f45a0948cbe19d47561f08ff5b12c253def5622a8f39b5bd640abbc44cb9f4ea02c219306ae36bef91f9b76976998bf6be12d8a6d1634bcef89ddfe
+  user: config.user
+  password: config.password
```
Comment (epintos) [inline config/secrets.yml]: Hmm, what is this `config.`?

File: Gemfile [hunk 2]
```diff
@@ -33,6 +33,10 @@ gem 'rspec'
 gem 'rubocop'
 
 gem 'devise'
+
+gem 'mandrill-api'
```
Comment (epintos) [inline Gemfile]: Pretty sure this gem is not necessary

File: config/secrets.yml (hunk 1 above)
Comment (rominariedel) [inline config/secrets.yml]: That references a .config.yml file that has the credentials and is included in the gitignore. It is managed by app_configuration gem

**Comment by epintos** [comment]
:+1:

**Pull request closed without merging**